examples/mqtt_client/mqtt_client.c

```diff
@@ -0,0 +1,249 @@
+
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+
+#include <string.h>
+
+#include <FreeRTOS.h>
+#include <task.h>
+#include <ssid_config.h>
+
+#include <espressif/esp_sta.h>
+#include <espressif/esp_wifi.h>
+
+#include <paho/MQTTESP8266.h>
+#include <paho/MQTTClient.h>
+
+#include <semphr.h>
+
```

Comment:
This example i wrote by myself. Files from baoshi's project doesn't have license headers, but whole project have a LICENSE.txt file. So i can add contents of LICENSE.txt file to each baoshi's file or just put whole file in extras/paho_mqtt_c. Which way is better?